examples/simple_repeater/main.cpp

```diff
@@ -247,6 +308,69 @@ class MyMesh : public mesh::Mesh, public CommonCLICallbacks {
       radio_set_params(_prefs.freq, _prefs.bw, _prefs.sf, _prefs.cr, _prefs.sync_word);
       MESH_DEBUG_PRINTLN("Radio params restored");
     }
+
+    #ifdef ENABLE_BLE
```

Comment:
style thing, `ifdef`'s and other macros used in the middle of code should not be tabbed in

Reply:
wait actually if `ENABLE_BLE` is not set, it will not run anything, plz fix

Reply:
Set by the variant's platformio build file, should work on all esp32 so it's at the top level `esp32_base`. Will enable it or disable it in places as we find out what's working.

https://github.com/datapartyjs/MeshTNC/pull/41/files#diff-4446afd728a4f34cbcddc306a9cb6be845d1a61c216076a295683bcc9c106724R60